src/PhpParser/Parser/ParserFactory.php

```diff
@@ -3,7 +3,7 @@
 namespace Rector\PhpParser\Parser;
 
 use PhpParser\Lexer;
-use PhpParser\Parser as NikicParser;
+use PhpParser\Parser;
```

Comment:
Wich Rector or CS was responsible for this one? :)

Reply:
```
bin/rector p src packages -l coding-style
```